src/Configuration/src/SpringBootBase/README.md

```diff
@@ -16,7 +16,7 @@ using Steeltoe.Extensions.Configuration.SpringBootEnv;
         static void Main(string[] args)
         {
             var host = Host.CreateDefaultBuilder()
-                        .ConfigureAppConfiguration(conf => conf.AddSpringBootEnvSource())
+                        .ConfigureAppConfiguration(conf => conf.AddSpringBootEnv())
```

Comment:
README does not reference the new CMD provider

Reply:
Updated readme.